### ISSUE TITLE
Generic expando object to handle search from front end MVC view

### ISSUE THREAD

**Issue by fasteddys** [baranacikgoz/PaginatedSearchAndFilter#3]
Hello, this is very nice after playing around I was hoping to get a sample with asp Core View, on how the developers can implement with a generic expando/dynamic object

- [ ] For e.g. if user are searching songs or images or some media, how can we get handle it as a generic request. 
- [ ] Also what do we need to configure in the pipeline request handler so it handle it.

**Comment by baranacikgoz** [comment]
> * [ ]  For e.g. if user are searching songs or images or some media, how can we get handle it as a generic request.

Unfortunately, I did not get what you mean.


> [ ] Also what do we need to configure in the pipeline request handler so it handle it.

Currently we have no plans for pipelines, but I'd love to hear your idea.

**Comment by fasteddys** [comment]
yes @baranacikgoz it would be even easier if we could annotate the attributes.

**Comment by fasteddys** [comment]
hello @baranacikgoz sorry about the confusion, hope you can visualize, I am posting some examples.

Usually theres a `poco `or `viewmodel`.. I have to attach search and pagination based on the results on the front end page

Some examples to help visualize, 
_______

![drex_searchandfilter_screen](https://github.com/baranacikgoz/PaginatedSearchAndFilter/assets/96149622/6fc353c1-4f16-4414-b89e-b891626a83c4)


_________

![Capture-233](https://github.com/baranacikgoz/PaginatedSearchAndFilter/assets/96149622/8d288a68-0885-4bba-a38c-3a4dc504f6ff)

__________

<img width="813" alt="uuid-750749cf-7051-2e99-412f-4663ca12b41e" src="https://github.com/baranacikgoz/PaginatedSearchAndFilter/assets/96149622/2f2831ff-6947-47cc-8227-dd334670ca2a">

**Comment by fasteddys** [comment]
hello
